client/dashboard/sites/overview/index.tsx

```diff
@@ -226,6 +227,10 @@ function SiteOverview( {
 				/>
 			}
 		>
+			<Survey
```

Comment:
`<Notice>`s needs to be passed as the `<PageLayout>`'s `notices` prop, not children, so that it renders with consistent spacing with the rest of MSD.

Reply:
Changed.